docs/headless-js-android.md

```diff
@@ -77,12 +120,32 @@ service.putExtras(bundle);
 getApplicationContext().startService(service);
 ```
 
+</TabItem>
+<TabItem value="kotlin">
+
+```kotlin
+val service = Intent(getApplicationContext(), MyTaskService::class.java)
```

Comment:
```suggestion
val service = Intent(applicationContext, MyTaskService::class.java)
```

The IDE should suggest you to simplify here. Can you confirm?

Reply:
Yes! You're right, thanks!